lib/galaxy/model/migrations/alembic/versions_gxy/c14a3c93d66a_add_user_defined_object_stores.py

```diff
@@ -0,0 +1,49 @@
+"""add user defined object stores
+
+Revision ID: c14a3c93d66a
+Revises: 460d0ecd1dd8
+Create Date: 2023-04-01 17:25:37.553039
+
+"""
+from alembic import op
+from sqlalchemy import (
+    Column,
+    DateTime,
+    ForeignKey,
+    Integer,
+    String,
+    Text,
+)
+
+from galaxy.model.custom_types import JSONType
+
+# revision identifiers, used by Alembic.
+revision = "c14a3c93d66a"
+down_revision = "3356bc2ecfc4"
+branch_labels = None
+depends_on = None
+
+
+# database object names used in this revision
+table_name = "user_object_store"
+
+
+def upgrade():
```

Comment:
Now that #15811 is merged, could you please use the `create_table`/`drop_table` utilities instead? (`model.migrations.util`). The function signatures are the same.

Reply:
Done thanks.